src/main/java/games/strategy/engine/framework/ArgParser.java

```diff
@@ -120,10 +120,42 @@ private boolean setSystemPropertyOrClientSetting(
   }
 
   private static boolean handleGameSetting(final String key, final String value) {
-    if (GameRunner.MAP_FOLDER.equals(key)) {
+    if (CliProperties.MAP_FOLDER.equals(key)) {
       ClientSetting.MAP_FOLDER_OVERRIDE.save(value);
       return true;
     }
     return false;
   }
+
+
+  /**
+   * An interface for all CLI related constants.
+   */
+  public interface CliProperties {
+    // argument options below:
+    static final String TRIPLEA_GAME_HOST_CONSOLE_PROPERTY = "triplea.game.host.console";
```

Comment:
Nit: `static` and `final` are redundant for interface fields.

One thing to consider...  We've been using interfaces internally (i.e. either private or package-private accessibility) for the purpose of defining constants simply because they are convenient.  However, the practice of using a _public or protected_ interface to define a group of constants is considered an anti-pattern because it may lead to classes outside the package implementing the interface, which leaks implementation details into the class's public API.

Item 19 of _Effective Java_ even goes on to say:

> There are several constant interfaces in the Java platform libraries, such as `java.io.ObjectStreamConstants`.  These interfaces should be regarded as anomalies and should not be emulated.

So, since `CliProperties` is a public type, it might be better to

* change it to a `public static final` nested class
* add a private constructor
* declare all fields as `public static final`